guidelines/sc/22/focus-appearance.html

```diff
@@ -5,11 +5,11 @@ <h4>Focus Appearance</h4>
     <p class="conformance-level">AA</p>
 	<p class="change">New</p>
     
-    <p>When the keyboard <a>focus indicator</a> is visible, one or both of the following is true:</p>
+    <p>When the keyboard <a>focus indicator</a> is visible, one or both of the following are true:</p>
     <ol>
-        <li>The focus indicator:
+        <li>The focus indicator meets all the following:
             <ul>
-                <li><a>encloses</a> the visual presentation of the user interface component, and</li>
+                <li><a>encloses</a> the <a>user interface component</a>  or sub-component that is focused, and</li>
```

Comment:
My concern here is due to things like cards where there are focusable sub-items and while the whole card is mouse-clickable the focus moves to the first item and hitting enter or clicking on that goes to the same place as clicking on the whole card. Sometimes it makes more sense to focus in on a smaller piece of a larger control.

Reply:
I emailed about this recently, but the card example isn't a sub-component.

The UIC in that case is the link, the thing that gets focus, and we can use the indicator to establish that. The clickable area isn't part of that analysis. There are several examples in the doc: https://docs.google.com/document/d/1P88HywmDChMklvKR_bwa2H4jW-CyJFj231HyPjjxY78/edit